src/Domain/Values/TypeValue.cs

```diff
@@ -0,0 +1,173 @@
+﻿namespace Erdcsharp.Domain.Values
+{
+    public class TypeValue
```

Comment:
`ProtocolType`?

Is this a combined class for all possible types? Numerical, structs, and so on?

Good it this works, but do you think you can add some documentation / code comments to explain your design?